src/server/location.rs

```diff
@@ -302,9 +304,13 @@ impl Location {
             Ok(location) => {
                 if let Some(names) = location.city.and_then(|c| c.names) {
                     result.city = get_preferred_language_element(&preferred_languages, &names)
+                } else {
+                    metrics.incr("location.unknown.city");
                 };
                 if let Some(names) = location.country.and_then(|c| c.names) {
                     result.country = get_preferred_language_element(&preferred_languages, &names)
+                } else {
```

Comment:
We could add these (country/subdivision) for the location header cases too

Reply:
Added the 'unknown' metrics to the google header code.